rest/route/github_routes.go

```diff
@@ -129,6 +129,9 @@ func (gh *githubHookApi) Execute(ctx context.Context, sc data.Connector) (Respon
 		} else if *event.Action == githubActionClosed {
 			return ResponseData{}, sc.AbortPatchesFromPullRequest(event)
 		}
+
+	case *github.PushEvent:
+		return ResponseData{}, sc.TriggerRepotracker(gh.queue, gh.msgID, event)
```

Comment:
can we put this in some sort of a feature flag? Until the cron repotracker stops running for projects that have pull request-based tracking enabled, I think this might not be safe.

Reply:
will add a degraded mode flag for this